charts/cluster/README.md

```diff
@@ -0,0 +1,11 @@
+UNDER DEVELOPMENT. DO NOT USE!
+
+## Things that are built but not tested. Remove corresponding lines after testing.
+
+* Azure Cloud Storage Backups
+* Google Cloud Storage Backups
+* Azure Cloud Storage Recovery
+* Google Cloud Storage Recovery
+
+## TODO
+* IAM Role for S3 Service Account 
```

Comment:
Do you have any plan yet about how to do this? I think it would be out of scope to actually set it up, but maybe we could hint at how to do that and provide the required configuration in the notes

Reply:
I've improved the README to include information on what's tested and what isn't along with a big warning message.

I've also included a note on the purpose of the chart based on yours and Jonathan's feedback in the CNPG meeting.

And finally, I've asked and @Otterpohl kindly agreed to look into the IAM Role functionality as I currently don't have the capability to do so.